.github/workflows/build-ampd-and-push-to-r2.yaml

```diff
@@ -48,7 +47,7 @@ jobs:
           commit_hash=$(git rev-parse --short HEAD)
           cd ..
           mkdir -p ./artifacts/$commit_hash/
-          sudo mv axelar-amplifier/artifacts/*.wasm ./artifacts/$commit_hash/
+          sudo mv axelar-amplifier/artifacts/* ./artifacts/$commit_hash/
```

Comment:
do you need sudo here ?

Reply:
@talalashraf yeah if fails otherwise. Take a look:
https://github.com/axelarnetwork/axelar-amplifier/actions/runs/9082686379/job/24959643182

Reply:
replaced `sudo mv` with `cp -R`